File: w1/PS 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
    int n;
    do {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);


    for (int row = 1; row <= n; row++) {
        for (int space = n-row; space > 0; space--) {
            printf(" ");
        }
        for (int hash = 1; hash <= row; hash++) {
            printf("#");
        }
        printf("  ");
        for (int hash2 = 1; hash2 <= row; hash2++) {
            printf("#");
        }
        printf("\n");
    }
}

File: w6/PS6/mario.py
def get_height():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0 and n < 9:
                break
        except ValueError:
            print("It must be  positive integer between 1 and 8, inclusive")
    return n

def mario():
    n = get_height()
    for row in range(1,n+1):
        for space in range(n-row,0,-1):
            print(" ", end="")
        for hash in range(1,row+1):
            print("#", end="")
        print("  ", end="")
        for hash2 in range(1,row+1):
            print("#", end="")
        print()

mario()